### PR TITLE
edit 'Considerations for file uploads' and 'Support for x-www-form-ur…

### PR DESCRIPTION
**Pull request by krishahn**
…lencoded request bodies'

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -1409,7 +1409,7 @@ requestBody:
             properties: {}
 ```
 
-Note that in the above example, the contents in the `requestBody` MUST be stringified per [RFC1866](https://tools.ietf.org/html/rfc1866/) when being passed to the server.  In addition, the `address` field complex object will be stringified as well.
+In this example, the contents in the `requestBody` MUST be stringified per [RFC1866](https://tools.ietf.org/html/rfc1866/) when being passed to the server.  In addition, the `address` field complex object will be stringified.
```
Comment (DavidBiesack) [inline versions/3.0.md]: consider dropping "being" as in "when passed to the server".

File: versions/3.0.md [hunk 2]
```diff
@@ -1360,9 +1360,9 @@ schema:
   format: binary
 ```
 
-Note that the above examples apply to either input payloads (i.e. file uploads) or response payloads.
+These examples apply to either input payloads of file uploads or response payloads.
 
-A `requestBody` example for submitting a file in a `POST` operation therefore may look like the following:
+A `requestBody` for submitting a file in a `POST` operation, however, may look like the following example:
```
Comment (DavidBiesack) [inline versions/3.0.md]: I this this should drop the "however".

File: versions/3.0.md (hunk 1 above)
Comment (krishahn) [inline versions/3.0.md]: +1

File: versions/3.0.md (hunk 2 above)
Comment (krishahn) [inline versions/3.0.md]: +1

**Pull request merged**